### PR TITLE
hasta specific changes

### PR DESCRIPTION
**Pull request by emiliaol**
* Remove seqtk directory
* Added sourcing of production conda env

### CODE REVIEW

File: scripts/downsample.sh [hunk 1]
```diff
@@ -6,7 +6,12 @@ set -e
 # import logging/versioning
 source /mnt/hds/proj/bioinfo/SCRIPTS/log.bash
 
-VERSION=1.4.1
+#activate production conda env
+shopt -s expand_aliases
+source ${HOME}/.bashrc
+useprod
+
+VERSION=1.4.2
```
Comment (ingkebil) [inline scripts/downsample.sh]: no need to bump the version yourself. We will bump as soon as we mere to master.

**Review by ingkebil — CHANGES_REQUESTED** [review]
Looks good! :package:

File: scripts/downsample.sh (hunk 1 above)
Comment (emiliaol) [reply scripts/downsample.sh]: Alrighty!

Comment (emiliaol) [reply scripts/downsample.sh]: How about now? :D

**Comment by emiliaol** [comment]
Tested after removing logging import:
Skärmavbild 2019-04-05 kl  14 54 01

Log info:
Skärmavbild 2019-04-05 kl  14 55 40

And the correct number of reads!
Skärmavbild 2019-04-05 kl  15 02 05

**Review by ingkebil — APPROVED** [review]
:clap:

**Pull request merged**